apps/trading-e2e/src/integration/auction-orders.feature

```diff
@@ -0,0 +1 @@
+Feature: Auction Orders
```

Comment:
For scenarios with auctions I think it makes more sense to group them with the other deal ticket tests. 
So I think this can be deleted.

Reply:
depending on how many tests in total there are in deal ticket it might be better to split them out in seperate feature files .
When running tests in parralel  you don't want the features to be too long to run